### DISCUSSION TITLE
Wrongly showing the time of the last reply as the time of creation in the list of discussions

### DISCUSSION THREAD

**Discussion by WrichikBasu** [community/community#2949] [Discussions] [answered]
Consider [this discussion](https://github.com/github/feedback/discussions/2861) in this repository:

![image](https://user-images.githubusercontent.com/56736644/101837709-7c6a2980-3b65-11eb-8b54-cce4b5bbe829.png)

Below the discussion title, it is written,

> agentpoyo started 10m ago in Discussions Feedback · Unanswered

That is not true! The discussion was started 2 days back. I posted the last comment 10 minutes back:

![image](https://user-images.githubusercontent.com/56736644/101837876-bdfad480-3b65-11eb-99d0-c908b4eb81d6.png)

It is wrong to show the date/time of the last reply as the time when the discussion was started.

Personally, I would prefer to see the date/time of the last reply instead of when the discussion was started.

**Comment by becca** [comment, marked as answer]
This has been fixed https://github.com/github/feedback/discussions/2855
